lib/compute_runtime/cli.rb

```diff
@@ -12,8 +12,9 @@ def run_rbwasm(*args)
       RubyWasm::CLI.new(stdout: $stdout, stderr: $stderr).run(args)
     end
 
-    def pack_directory(infile, outfile, dirmaps)
+    def pack_directory(infile, outfile, dirmaps, dirmap_strings)
       mapdir_args = dirmaps.collect_concat { |k, v| ["--mapdir", "#{k}::#{v}"] }
```

Comment:
```suggestion
      mapdir_args = dirmaps.collect_concat { |k, v| ["--dir", "#{v}::#{k}"] }
```

It's not a part of your change, but could you update here also?